cli/runtime/config/nodeutils/helpers.go

```diff
@@ -18,3 +23,21 @@ func UniqNodes(nodes []*yaml.Node) []*yaml.Node {
 
 	return uniq
 }
+
+// equalScalars returns true if two scalar nodes has same value
+func equalScalars(left, right *yaml.Node) (bool, error) {
+	if left.Kind == yaml.ScalarNode && right.Kind == yaml.ScalarNode {
+		return left.Value == right.Value, nil
+	}
+	return false, errors.New("equals on non-scalars not implemented")
+}
+
+func checkErrors(src, dst *yaml.Node) error {
+	if src.Kind != dst.Kind {
+		return ErrDifferentArgumentsTypes
+	}
+	if dst != nil && reflect.ValueOf(dst).Kind() != reflect.Ptr {
```

Comment:
we don't need to check dst as `reflect.Ptr` as a function accepts only pointers

Reply:
removed